cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -378,6 +379,9 @@ type ControllerContext struct {
 
 	// ControllerManagerMetrics provides a proxy to set controller manager specific metrics.
 	ControllerManagerMetrics *controllersmetrics.ControllerManagerMetrics
+
+	// GarbageCollector provides access to the GC instance for use in other controllers.
+	GarbageCollector *garbagecollector.GarbageCollector
```

Comment:
you don't need a garbage collector.  You need an interface on the GraphBuilder, right?